dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipsExportControllerTest.java

```diff
@@ -245,11 +246,28 @@ void getRelationshipsByEvent() {
             .getList("instances", JsonRelationship.class);
 
     JsonObject relationship = assertFirstRelationship(r, relationships);
-    assertHasOnlyMembers(relationship, "relationship", "relationshipType", "from", "to");
+    assertHasOnlyMembers(
+        relationship, "relationship", "relationshipType", "createdAtClient", "from", "to");
     assertHasOnlyUid(from.getUid(), "event", relationship.getObject("from"));
     assertHasOnlyUid(to.getUid(), "trackedEntity", relationship.getObject("to"));
   }
 
+  @Test
+  void getRelationshipsByEventWithAllFields() {
```

Comment:
are we validating anything related to the field `createdAtClient` in this test?

Reply:
Yes, we are asserting the relationship and in there we are checking that `createdAtClient` is the expected one